### ISSUE TITLE
[Feature Request] - Individual Rule Metrics

### ISSUE THREAD

**Issue by tdarwin** [honeycombio/refinery#631]
**Describe the solution you'd like**
Having a conversation with a client some time ago, they wanted to try to figure out what percentage of traces sent to honeycomb were coming through each rule.  We don't have metrics for each rule created, so it's hard to parse that against the total `trace_send_kept` value.

It would be cool if we could auto-generate some metric keys for `trace_accepted_<dataset>_<rule_name>`, `trace_send_kept_<dataset>_<rule_name>`, and `trace_send_dropped_<dataset>_<rule_name>` so that those could be used for even more refinement or just app insights to get kept/send rule statistics from the metrics refinery is sending, rather than just through comparing Usage and Normal mode query results.

**Comment by kentquirk** [comment]
Great idea. I think we can look at this when we tackle #624.

**Comment by MasonLegere** [comment]
Should this be closed following https://github.com/honeycombio/refinery/pull/714 or is the `dataset` being included in the metric considered important? In a world of microservices I think it could end up being a lot of metrics so may have to be opt-in anyways i.e. 30 services * ~5 samplers potentially * # of clusters with refinery instances producing separately tagged metrics
